modules/order/src/Plugin/Field/FieldType/AdjustmentItem.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\Field\FieldType;
+
+use Drupal\commerce_order\Adjustment;
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Plugin implementation of the 'commerce_adjustment' field type.
+ *
+ * @FieldType(
+ *   id = "commerce_adjustment",
+ *   label = @Translation("Adjustment"),
+ *   description = @Translation("Stores an adjustment which contains a decimal amount and a three letter currency code and other metadata."),
```

Comment:
Doesit matter what the adjustment contains? "Stores adjustments to the parent entity's price."